bootstrap/bpdoc/bpdoc.go

```diff
@@ -173,10 +173,14 @@ func nestedPropertyStructs(s reflect.Value) map[string]reflect.Value {
 				// The field is not exported so just skip it.
 				continue
 			}
+			if hasTag(field.Tag, "blueprint", "mutated") {
+				continue
+			}
 
 			fieldValue := structValue.Field(i)
+			kind := fieldValue.Kind()
```

Comment:
Why this change?

Reply:
Leftover from debugging endeavor. Removed.